backup/crowbar-backup

```diff
@@ -88,12 +88,12 @@ function check_resolv_conf
 {
   # check if DNS servers are available and can resolve domain names
   for ns in `grep "^\s*nameserver" /etc/resolv.conf | sed -e "s/\s*nameserver \([[:digit:]\.]\+\)[^[:digit:]\.]*/\1/" ` ; do
-    if ! netcat -z $ns 53 ; then
+    if ! netcat -zu $ns 53 ; then
```

Comment:
from man netcat : 
CAVEATS

UDP port scans using the -uz combination of flags will always report success irrespective of the target machine's state. However, in conjunction with a traffic sniffer either on the target machine or an intermediary device, the -uz combination could be useful for communications diagnostics. Note that the amount of UDP traffic generated may be limited either due to hardware resources and/or configuration settings.